docs/products/clickhouse/howto/manage-databases-tables.md

```diff
@@ -76,6 +76,11 @@ Aiven for ClickHouse uses `replicated` variants of table
 engines to ensure high availability. Even if you select `MergeTree`
 engine, we will automatically use the replicated variant on our side.
 
+:::note
+A query log is a non-replicated table. Running a query again a query log requires
+[sending a request to all replica nodes](/docs/products/clickhouse/howto/query-databases#query-a-query-log).
+:::
```

Comment:
I would say that for querying non replicated tables, (like query log) clusterAllReplicas might be used, and for shards - distributed table might be used.